app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    content = command.split(" ")
+    if content[1] != content[2]:
+        file_one = content[1]
```

Comment:
The variable names `file_one` and `file_two` are not very descriptive. Consider using more descriptive names like `source_file` and `destination_file`.